lib/matplotlib/quiver.py

```diff
@@ -173,6 +174,15 @@
 
     'mid' is a synonym for 'middle'.
 
+head_pos : {'tail', 'mid', 'middle', 'tip'}, optional, default: 'tip'
```

Comment:
I would suggest having a single head_pos argument that takes either a string or a float, with "tail" synonymous to 0, "middle" synonymous to 0.5, and tip synonymous to 1 (or reverse 0 and 1 -- it's actually not clear from the docstring which is which).
Also perhaps the pivot kwarg should also gain the same functionality?

Reply:
I was following the style of pivot for similarity within the module. But in principle, I agree with you both should probably be more flexible. I am not sure however if the angles become significantly more complex if the pivot parameter is let loose.

Reply:
I have made the edits to head_pos and committed changes I have not edited pivot but this could be opened as a feature request should people actually want it.